iotdb/src/test/java/org/apache/iotdb/db/integration/IoTDBMetadataFetchIT.java

```diff
@@ -374,8 +374,9 @@ private void showTimeseriesInJson() {
             + "                temperature:{\n"
             + "                     DataType: FLOAT,\n"
             + "                     Encoding: RLE,\n"
-            + "                     Compressor: SNAPPY,\n"
-            + "                     args: {max_point_number=3},\n"
+            + "                     Compressor: UNCOMPRESSED,\n"
+            // FIXME; something seems strange here
+            + "                     args: {max_point_number=3, compressor=SNAPPY},\n"
```

Comment:
Compressor = UNCOMPRESSED and compressor=SNAPPY?

Reply:
It is incorrect because I pick max_point_number out of args

Reply:
fixed